payjoin/src/receive/v1/mod.rs

```diff
@@ -829,7 +831,7 @@ pub(crate) mod test {
             .commit_outputs()
             .commit_inputs();
 
-        let payjoin = payjoin.apply_fee(None, FeeRate::ZERO);
+        let payjoin = payjoin.apply_fee(None, Some(FeeRate::ZERO));
```

Comment:
Should this not now take advantage of the default by passing `None`?

I feel like our reference should take advantage of our sane defaults if we believe them to be sane.